sdk/communication/AzureCommunicationChat/Tests/ChatThreadClientUnitTests.swift

```diff
@@ -716,6 +720,34 @@ class ChatThreadClientUnitTests: XCTestCase {
         }
     }
 
+    func test_SendTypingNotification_WithDisplayName_ReturnSuccess() {
```

Comment:
These seem like end-to-end tests rather than unit tests.

Reply:
there is an E2E test as well, unit test covers calls with + without the sender displayname param